src/rich-text-editor.component.js

```diff
@@ -7,9 +7,8 @@ let globalBandicootId = 0
 export const RichTextEditor = forwardRef((props, editorRef) => {
   const divRef = useRef(null)
   const selectionRangeBeforeBlurRef = useRef(null)
-  const [ focused, setFocused ] = useState(() => false)
+  const [ focused, setFocused ] = useState(() => null)
```

Comment:
if this is null, then we won't enter 64 on first load which is important. No need to `fireBlur` or `save`

Reply:
fyi, not important: `() => null` is worse than `null` for performance, although the performance impact is so negligible that it doesn't matter in any real way. The creation of an inline function is more expensive than just passing in null.